streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -544,6 +563,11 @@
 
             // MEDIUM
 
+            .define(ASSIGNMENT_ACCEPTABLE_RECOVERY_LAG_CONFIG,
```

Comment:
Put this one as "medium" importance and the rest as low, because this seems like the most likely config that a typical user might need to tweak for their specific app/workload